internal/transform/account.go

```diff
@@ -0,0 +1,93 @@
+package transform
+
+import (
+	"fmt"
+
+	"github.com/stellar/go/xdr"
+)
+
+//TransformAccount converts an account from the history archive ingestion system into a form suitable for BigQuery
+func TransformAccount(ledgerEntry xdr.LedgerEntry) (AccountOutput, error) {
+	accountEntry, accountFound := ledgerEntry.Data.GetAccount()
+	if !accountFound {
+		return AccountOutput{}, fmt.Errorf("Could not extract account data from ledger entry; actual type is %s", ledgerEntry.Data.Type)
+	}
+
+	outputID, err := accountEntry.AccountId.GetAddress()
+	if err != nil {
+		return AccountOutput{}, err
+	}
+
+	outputBalance := int64(accountEntry.Balance)
+	if outputBalance < 0 {
+		return AccountOutput{}, fmt.Errorf("Balance is negative (%d) for account: %s", outputBalance, outputID)
+	}
+
+	accountExtentionInfo, V1Found := accountEntry.Ext.GetV1()
```

Comment:
Worth adding a comment here explaining what the `V1` struct contains. In the case of a bug in this section, that wouldn't be immediately obvious, and the information would help debugging.